### PR TITLE
Dala 4490 keep nace code empty in frontend

### PR DESCRIPTION
**Pull request by dennis-dfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [x] If this PR includes work on the frontend, at least one `@ts-nocheck` is removed. Additionally, there should not be any `@ts-nocheck` in files modified by this PR. If no `@ts-nocheck` are left: Celebrate :tada: :confetti_ball: type-safety and remove this entry. 
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team

### CODE REVIEW

File: gitinfo
Comment (jhaerDfine) [inline gitinfo:1]: changes in der file nicht mit committen

File: dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue
Comment (jhaerDfine) [inline dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue:1]: laut pr template soll '@ts-nocheck' in files die bearbeitet werden removed werden (und dann müssen vermutlich ein paar sachen angepasst werden).

File: gitinfo
Comment (dennis-dfine) [reply gitinfo:1]: resolve. hab den commit reverted und nur die changes im ActivitySelecter gepushed

File: dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue [hunk 1]
```diff
@@ -43,13 +43,14 @@
       ref="multiSelectFormFieldRef"
       dataTest="selectNaceCodes"
       name="naceCodes"
-      :emptyMessage="`No NACE code available for ${selectedActivities.name} Activity`"
-      :validation="naceCodesForActivities.length ? 'required' : ''"
```
Comment (LaurinStrelow) [inline dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue:47]: Removing this line fixes the problem, and the remaining changes are typescript related?

Comment (dennis-dfine) [reply dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue:47]: Yes. Zeile 47 hier ist der eigentliche change. Alles andere is fix-up

File: dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue [hunk 2]
```diff
@@ -43,13 +43,14 @@
...
+      :emptyMessage="`No NACE code available for ${selectedActivities?.name} Activity`"
```
Comment (LaurinStrelow) [inline dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue:46]: Can selectedActivities be null here?

Comment (dennis-dfine) [reply dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue:46]: Klar, insbesondere, wenn du die Liste der Activities neu aufmachst, ist ja nix vorausgewählt.

**Pull request merged**